ori/testing/volume/fake.go

```diff
@@ -118,6 +119,20 @@ func (r *FakeRuntimeService) CreateVolume(ctx context.Context, req *ori.CreateVo
 	}, nil
 }
 
+func (r *FakeRuntimeService) ExpandVolume(ctx context.Context, req *ori.ExpandVolumeRequest, opts ...grpc.CallOption) (*ori.ExpandVolumeResponse, error) {
+	r.Lock()
+	defer r.Unlock()
+
+	volume, ok := r.Volumes[req.VolumeId]
+	if !ok {
+		return nil, fmt.Errorf("volume with id %s not found", req.VolumeId)
```

Comment:
Return a `codes.NotFound` error here, similar to the other functions of the fake runtime.